pandas/tseries/offsets.py

```diff
@@ -1829,13 +1835,14 @@ class QuarterOffset(DateOffset):
     # TODO: Consider combining QuarterOffset and YearOffset __init__ at some
     #       point
 
-    def __init__(self, n=1, normalize=False, **kwds):
+    def __init__(self, n=1, normalize=False, startingMonth=None):
```

Comment:
can you make an issue that this should be deprecated to ``starting_month`` (and prob others like this)